### DISCUSSION TITLE
suggestion about information and colors rectangles

### DISCUSSION THREAD

**Discussion by Phil353556** [jvde-github/AIS-catcher#265] [Ideas]
Hi,
Is it possible, if not too difficult to enhanced the information and add the station number for each rectangle, when the pointer is over?

-red rectangle:  so the station number is the station itself,
-orange rectangle: two station numbers to indicate,
-green rectangle: more than three station numbers to indicate

Thank you
Phil
![image](https://github.com/jvde-github/AIS-catcher/assets/64729485/fceedc02-ceb0-44cb-afd6-a720977f7621)

**Comment by jvde-github** [comment]
![image](https://github.com/jvde-github/AIS-catcher/assets/52420030/909b0ce4-e85a-4675-84a6-31b959e0dedc)

Added. Might have to speed up the request a bit at some point though. Note this is over 72 hour period and contains a lot of lower station numbers from anonymous feeders (which are somewhat fixed but feeders can show up with several station ids)

**Reply by jvde-github** [reply]
Note: also fixed a bug in the calculation of the message count.

**Comment by Phil353556** [comment]
Thank you so much Jasper @jvde-github !
